brainscore_vision/benchmarks/hebart2023/test.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from brainscore_vision import benchmark_registry, load_model
+from brainscore_vision.benchmarks.hebart2023 import Hebart2023Accuracy
+
+def test_benchmark_registry():
+    assert "Hebart2023-acc" in benchmark_registry
+
+def test_ceiling():
+    benchmark = Hebart2023Accuracy()
+    ceiling = benchmark.ceiling
+    assert ceiling.sel(aggregation='center') == pytest.approx(0.6844, abs=0.0001)
+
+
+# This test takes longer than 10 minutes. See also:
+# https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received
+# def test_alexnet_consistency():
+#     benchmark = Hebart2023Accuracy()
+#     model = load_model('alexnet')
+#     score = benchmark(model)
+#     assert score.sel(aggregation='center') == pytest.approx(0.38, abs=0.02)
```

Comment:
This takes way longer than 10 minutes. Should we maybe create a sub stimulus set for testing only @mschrimpf @kvfairchild?